core/jazz_create-serverless-service/index.js

```diff
@@ -351,6 +351,19 @@ var getServiceData = (service_creation_data, authToken, configData, deploymentTa
     if (service_creation_data.service_type === "api" || service_creation_data.service_type === "function" || service_creation_data.service_type === "sls-app") {
       serviceMetadataObj.providerRuntime = service_creation_data.runtime;
     }
+    
+    //Adding providerTimeout key in service catalog
+    if (service_creation_data.service_type === "api" || service_creation_data.service_type === "function") {
+      if (service_creation_data.timeout)
```

Comment:
User won't provide the timeout value during service creation. We can skip this check.

Reply:
Updated.